js/src/get-started-page/case-studies-card/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Card,
+	CardHeader,
+	Flex,
+	FlexBlock,
+	__experimentalText as Text,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './index.scss';
+import quoteImageURL from './img-quote.svg';
+
+const ContentBlock = ( { content, name } ) => {
+	return (
+		<FlexBlock>
+			<img
+				src={ quoteImageURL }
+				alt={ __( 'An image of a quote.', 'google-listings-and-ads' ) }
+				width="34"
+				height="34"
+			/>
+			<Text
+				className="gla-get-started-case-studies-card__content"
+				variant="body"
+			>
+				{ content }
+			</Text>
+			<Text className="gla-get-started-case-studies-card__name">
+				{ name }
+			</Text>
+		</FlexBlock>
+	);
+};
+
+const CaseStudiesCard = () => {
+	return (
+		<Card className="gla-get-started-case-studies-card" isBorderless>
+			<CardHeader>
+				<Text
+					className="gla-get-started-case-studies-card__title"
+					variant="title.medium"
+				>
+					{ __(
+						'21,000+ WooCommerce store owners like you already list products on Google for free',
+						'google-listings-and-ads'
+					) }
+				</Text>
+			</CardHeader>
+			<Flex gap={ 0 }>
+				<ContentBlock
```

Comment:
💅  I'm not convinced neither by ContentBlock as name for this. Maybe `TestimonialItem` or `Quote` or `Testimonial`?

Reply:
Thanks for the suggestion! Updated in 6404822.